ERROR:root:Traceback (most recent call last):
  File "E:/AppiumTest/Cam/TEMP.py", line 89, in <module>
    error_func()
  File "E:/AppiumTest/Cam/TEMP.py", line 84, in error_func
    b = 1 / 0
ZeroDivisionError: division by zero

